ding_pretix.module

```diff
@@ -73,6 +73,11 @@ function ding_pretix_field_formatter_info() {
 
 /**
  * Implements hook_node_insert().
+ *
+ * @param $node
+ *   The node that is being created.
+ *
+ * @throws Exception if one of the INSERTs failed
```

Comment:
The exception type should be defined if possible and comment on next line. See https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#throws

Reply:
This is true for all @throws in the code